Ninjathulhu/Ninjathulhu.Game/Component.cs

```diff
@@ -20,7 +17,7 @@ class Component
 
         public virtual void Start() {}
 
-        public virtual void Update(float elapsed) {}
```

Comment:
Yes, I realized this after I had checked in it.  Probably this should be Tick() rather than Update(), since we're planning a "pure" rogue-like without a realtime element.